libs/common-utils/src/environments.ts

```diff
@@ -1,7 +1,21 @@
 import { registerOnWindow } from './misc'
 
-const getRegex = (regex: string | undefined) => (regex ? new RegExp(regex, 'i') : undefined)
+const DEFAULT_ENVIRONMENTS_REGEX: Record<Envs, string> = {
+  LOCAL: '^(:?localhost:d{2,5}|(?:127|192)(?:.[0-9]{1,3}){3})',
+  PR: '^prd+--cowswap.review|^(swap-dev-git-[wd-]+|swap-w{9}-)cowswap.vercel.app',
```

Comment:
No longer needed, it was the old gnosis pr review link

```suggestion
  PR: '^(swap-dev-git-[wd-]+|swap-w{9}-)cowswap.vercel.app',
```

Reply:
https://github.com/cowprotocol/cowswap/pull/3309